src/DynamoApplications/StartupUtils.cs

```diff
@@ -0,0 +1,302 @@
+            var rootFolder = Path.GetDirectoryName(exePath);
+
+            var versions = new[]
+            {
+                LibraryVersion.Version219,
+                LibraryVersion.Version220,
+                LibraryVersion.Version221
+            };
+
+            var preloader = new Preloader(rootFolder, versions);
+            preloader.Preload();
+            geometryFactoryPath = preloader.GeometryFactoryPath;
+            preloaderLocation = preloader.PreloaderLocation;
+        }
+
+        public static DynamoModel MakeModel(bool CLI, IEnumerable<string> librariesToPreload)
+        {
+
+            var geometryFactoryPath = string.Empty;
+            var preloaderLocation = string.Empty;
+            PreloadShapeManager(ref geometryFactoryPath, ref preloaderLocation);
+
+            var config = new DynamoModel.DefaultStartConfiguration()
+                  {
+                      PathResolver = new PathResolver(preloaderLocation, librariesToPreload),
+                      GeometryFactoryPath = geometryFactoryPath,
+                      StartInTestMode = true
+                  };
+
+            //if we are building a model for CLI mode, then we don't want to start an updateManager
+            //for now, building an updatemanager instance requires finding Dynamo install location
+            //which if we are running on mac os or *nix will use different logic then SandboxLookup 
+
+            UpdateManagerConfiguration umConfig = null;
+            if (!CLI)
+            {
+                umConfig = UpdateManagerConfiguration.GetSettings(new SandboxLookUp());
+                Debug.Assert(umConfig.DynamoLookUp != null);
+                config.UpdateManager = new Dynamo.UpdateManager.UpdateManager(umConfig);
+                config.StartInTestMode = false;
+            }
+
+            var model = DynamoModel.Start(config);
+            return model;
+        }
+
+        public static List<string> SandBoxLibraries
+        {
+            get
+            {
+                return new List<string> 
+                {
+                "VMDataBridge.dll",
+                "ProtoGeometry.dll",
+                "DSCoreNodes.dll",
+                "DSOffice.dll",
+                "DSIronPython.dll",
+                "FunctionObject.ds",
+                "Optimize.ds",
+                "DynamoConversions.dll",
+                "DynamoUnits.dll",
+                "Tessellation.dll",
+                "Analysis.dll"
+                };
+
+            }
+        }
+
+        public static List<string> CLILibraries
+        {
+            get
+            {
+                return new List<string> 
+                {
+                //"ProtoGeometry.dll",
+                //"DSCoreNodes.dll",
+                //"DSIronPython.dll",
+                //"FunctionObject.ds",
+                //"VMDataBridge.dll",
+                };
+
+            }
+        }
+
+    }
+
+    public static class Locale
```

Comment:
Let's put all of these utility methods in the `StartupUtils` class. They just get a bit wordy when you have to say `Dynamo.Applications.StartupUtils.Preloading.CLILibraries`. Let's just have:

`Dynamo.Applications.StartupUtils.SetLocale`
`Dynamo.Applications.StartupUtils.MakeDynamoModel`
`Dynamo.Applications.StartupUtils.PreloadShapeManager`
`Dynamo.Applications.StartupUtils.DefaultSandboxLibraries`